.github/workflows/ci.yml

```diff
@@ -194,3 +194,40 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           files: lcov.info
+
+  std-tests:
+    env:
+      NUSHELL_CARGO_TARGET: ci
+
+    strategy:
+      fail-fast: true
+      matrix:
+        platform: [windows-latest, macos-latest, ubuntu-20.04]
+        style: [default, dataframe]
+        rust:
+          - stable
+        include:
+          - style: default
+            flags: ""
+          - style: dataframe
+            flags: "--features=dataframe"
```

Comment:
Mhh that might be a limitation on the virtualenv front as we don't run dataframe their anymore.

Reply:
to be honest with you, i did not really look at the whole CI and just copied the `nu-tests` job and modified the minimum, i.e. the name, the `cargo install` above and the `nu .../tests.nu` :confused:

maybe that's a change to apply on the whole CI then ? :open_mouth:

Reply:
I think we should bite the bullet and add it for the ubuntu runner if necessary.

Reply:
i'm not sure i understand what should be done here :open_mouth: